rubric_data/management/commands/generate_test_data.py

```diff
@@ -0,0 +1,72 @@
+from django.core.management.base import BaseCommand
+
+import argparse
+import json
+import random
+import string
+
+class Command(BaseCommand):
+    help = 'Scrubs real data of sensitive information'
+
+    def add_arguments(self, parser):
+        parser.add_argument("infile", type=argparse.FileType('r'), help="JSON file to be scrubbed")
```

Comment:
I didn't know about the `argparse.FileType`, that is cool.

Reply:
Yeah, its much nicer than using `open` and sanitizing the file path inputs from users.